Python/sysmodule.c

```diff
@@ -1732,9 +1732,13 @@ static Py_ssize_t
 sys_getallocatedblocks_impl(PyObject *module)
 /*[clinic end generated code: output=f0c4e873f0b6dcf7 input=dab13ee346a0673e]*/
 {
+    #if PY_DEBUGGING_FEATURES
```

Comment:
I'm curious, what's the link error here?

Reply:
sys_getallocatedblocks_impl gets referenced later despite being ifdefed out. I stubbed to remove the error instead of removing the reference since its a documented member of sys

Reply:
I'm confused about what's going on here, is MSVC trying to compile ifdef'd out code? What's the error you get?

I removed the function because I believe it's better to not have the function than to have it behave incorrectly. It looks like the removal was unnecessary, so if you want you could just remove the `PY_DEBUGGING_FEATURES` ifdefs.